### PR TITLE
initial POC dataset join and upload

### PR DESCRIPTION
**Pull request by hunterowens**
This PR adds the initial ETL for the Google Docs Shelter Occupancy Later. 

It downloads the data from google, merges it with static geojson download from LASAN on locations, can then uploads it to lahub. 

Improvements we should make 

* add tier 2 shelters 
* verify each of the names match for the string join 
* move to live shelter layers vs static

### CODE REVIEW

File: dags/public-health/covid19/shelter-to-esri.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+TIER_2_META_URL = "https://services1.arcgis.com/X1hcdGx5Fxqn4d0j/ArcGIS/rest/services/COVID_19_Shelters_Tier2/FeatureServer/0/query?where=1%3D1&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&resultType=none&distance=0.0&units=esriSRUnit_Meter&returnGeodetic=false&outFields=&returnGeometry=true&featureEncoding=esriDefault&multipatchOption=xyFootprint&maxAllowableOffset=&geometryPrecision=&outSR=&datumTransformation=&applyVCSProjection=false&returnIdsOnly=false&returnUniqueIdsOnly=false&returnCountOnly=false&returnExtentOnly=false&returnQueryGeometry=false&returnDistinctValues=false&cacheHint=false&orderByFields=&groupByFieldsForStatistics=&outStatistics=&having=&resultOffset=&resultRecordCount=&returnZ=false&returnM=false&returnExceededLimitFeatures=true&quantizationParameters=&sqlFormat=none&f=pgeojson&token="
+
+SHELTER_CSV_URL = "https://docs.google.com/spreadsheets/d/1pkg7PVCS4lwVhNA3TkMSWKmzw4vsd4W53_Q_Ou2mXqw/export?format=csv&id=1pkg7PVCS4lwVhNA3TkMSWKmzw4vsd4W53_Q_Ou2mXqw&gid=73455498"
```
Comment (ian-r-rose) [inline dags/public-health/covid19/shelter-to-esri.py]: Is this sheet open?

File: dags/public-health/covid19/shelter-to-esri.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+SHELTER_ID = "282aff8ca27447c8b4e7792671bc9dc3"
+def load_data(**kwargs):
+    """
+    Entry point for the DAG, loading shelter to ESRI.
+    """
+    gdf = gpd.read_file('./tier-1.geojson')
```
Comment (ian-r-rose) [inline dags/public-health/covid19/shelter-to-esri.py]: I hate to keep harping on this, and we really need to fix it, but I don't know if the geopandas install in aqueduct is currently working. We at least need to check on it.

File: dags/public-health/covid19/shelter-to-esri.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    df = pd.read_csv(SHELTER_CSV_URL)
+    df = df.rename({'Shelter Site? ': 'ParksName'}, axis = 1)      
+
+    # need to string match join here, which is a pain. 
+
+    df = df.replace(to_replace='109th Recreation Center', value='109th Street Recreation Center')
```
Comment (ian-r-rose) [inline dags/public-health/covid19/shelter-to-esri.py]: Ouch

File: dags/public-health/covid19/shelter-to-esri.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    # merge on parksname
+    # this preserves the number of entries (tested.)
+    gdf = gdf.merge(df, on='ParksName') 
+
+    # export to CSV
+    gdf['Latitude'] = gdf.geometry.x
+    gdf['Longitude'] = gdf.geometry.y
+    time_series_filename = "/tmp/shelter_timeseries_current.csv"
+
+    pd.DataFrame(gdf).to_csv(time_series_filename, index=False)
```
Comment (ian-r-rose) [inline dags/public-health/covid19/shelter-to-esri.py]: What does this do to the `Point` column? Convert to WKT, or fail? We may want to drop it.

File: dags/public-health/covid19/shelter-to-esri.py (hunk 2 above)
Comment (ian-r-rose) [reply dags/public-health/covid19/shelter-to-esri.py]: Is this intended to be replaced by the live URLs above?

Comment (hunterowens) [reply dags/public-health/covid19/shelter-to-esri.py]: yeah that would be my hope. need to figure out how to use [this](https://github.com/openaddresses/pyesridump) to geopandas inside a script

File: dags/public-health/covid19/shelter-to-esri.py (hunk 4 above)
Comment (hunterowens) [reply dags/public-health/covid19/shelter-to-esri.py]: adding commit to drop.

File: dags/public-health/covid19/shelter-to-esri.py (hunk 2 above)
Comment (ian-r-rose) [reply dags/public-health/covid19/shelter-to-esri.py]: Is there a reason not to construct the URL query with `objectid=objectid` and GeoJSON output? Avoids another dependency...

**Pull request merged**